active/social-media.md

```diff
@@ -0,0 +1,72 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability by generating revenue through sponsor relationships/campaigns.
```

Comment:
```suggestion
- Protect the framework's long-term viability by generating revenue through sponsor relationships/campaigns.
- Assist the board in posting official news and updates from DSF.
- Maintain and monitor automation in connection with posting DSF news and updates to social media channels.
- Enforce the Django Code of Conduct on Django's social media channels, such as deleting comments that do not meet our CoC or blocking users that grossly repeatedly violate the CoC.
```

I'm not sure exactly how to phrase this. But it's basically supposed to cover the "status quo".

Oh and then I added the CoC thing :)

Reply:
Ah that's a really good point, thank you!